Bot/bot.py

```diff
@@ -35,8 +35,7 @@
 client = commands.Bot(command_prefix='!cov ')
 client.remove_command('help')
 
-#session = aiohttp.ClientSession()
-
+api_client = diseaseapi.Client().covid19
 
 @client.command(pass_context=True, help="Restarts the bot, requires admin permission")

```

Comment:
> noticed a things looking quickly through it which would be worth changing, in particular replacing `all_countries()` with `country()`

Yes i did find that method, however to support short forms for country names like say `in` for `India`, it would require a loop through all the possible countries to find the closest match. Directly searching for `in` would throw an exception

Regarding `pass_context=True`, it doesn't impact performance and improves readability is what i think. Sorry if i am wrong